packages/orchestration/src/orchestration-api.ts

```diff
@@ -0,0 +1,109 @@
+/**
+ * @file Public-facing API of orchestration.
+ * - must not depend on Cosmos types in anyway.
```

Comment:
well, it depends on `./types.js`, which depends on Cosmos types such as `MsgBeginRedelegateResponse` by way of `cosmos-api`.

Reply:
Are you finding that the types exported by this module depend on Cosmos types? I didn't yet attempt to make the entire modules layered, just the APIs they describe.

Reply:
a TODO would make it more clear.

It's a `@file` comment, and it says "**must** not depend on Cosmos type **in anyway**".